examples/error-states.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="UTF-8">
+  <title>Booking.js error states examples</title>
+  <style type="text/css">
+    body { background-color: #E6E6E6; max-width: 700px; margin: 0 auto; }
+  </style>
+ </head>
+ <body>
+
+  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <script type="text/javascript" src="../dist/booking.js"></script>
+
+  <p>No email:</p>
+  <div id="bookingjs1">
```

Comment:
This `id` is quite generic. Should is this for ease of targeting? I couldn't find the reference for it.